psi4/driver/qcdb/libmintsbasisset.py

```diff
@@ -715,7 +715,12 @@ def construct(cls, parser, mol, key, deffit=None, basstrings=None, return_atomli
         mol.update_geometry()
 
         # Paths to search for gbs files: here + PSIPATH + library
-        psidatadir = os.environ.get('PSIDATADIR', None)
+        try:
+            from psi4 import core
+            psidatadir = core.get_datadir()
+        except ImportError:
+            pass
```

Comment:
Doesn't this allow the possibility for `psidatadir` to be undefined?

Reply:
Yes, but only in QCDB standalone mode and only for the library part of the search path. Smooth sailing (no envvar config) not guaranteed outside of P4.